madmom/audio/cepstrogram.py

```diff
@@ -121,6 +125,22 @@ def process(self, data, **kwargs):
 MFCC_NORM_FILTERS = True
 MFCC_MUL = 1.
 MFCC_ADD = np.spacing(1)
+MFCC_DCT_NORM = "ortho"
+MFCC_DELTA_FILTER = np.linspace(4, -4, 9) / 60
+MFCC_DELTADELTA_FILTER = np.linspace(1, -1, 3) / 2
+
+
+# https://stackoverflow.com/questions/3012421/python-memoising-deferred-lookup-property-decorator#3013910
+def lazyprop(fn):
+    attr_name = '_lazy_' + fn.__name__
+
+    @property
+    def _lazyprop(self):
+        if not hasattr(self, attr_name):
+            setattr(self, attr_name, fn(self))
+        return getattr(self, attr_name)
+
+    return _lazyprop
```

Comment:
This decorator can go into utils since it is rather useful. Please document it, though.